composer.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "0.1.0",
+  "name": "badaso/postfree-theme",
+  "description": "Official free theme for post module system on badaso",
+  "keywords": [
+    "laravel",
+    "admin",
+    "panel",
+    "dashboard",
+    "pwa",
+    "generator",
+    "blog",
+    "badaso",
+    "news"
+  ],
+  "license": "proprietary",
+  "homepage": "https://badaso.uatech.co.id/",
+  "support": {
+    "issues": "https://github.com/uasoft-indonesia/badaso-postfree-theme/issues",
+    "discussion": "https://github.com/uasoft-indonesia/badaso-postfree-theme/discussions",
+    "source": "https://github.com/uasoft-indonesia/badaso-postfree-theme"
+  },
+  "type": "library",
+  "require": {
+    "symfony/var-exporter": "*",
+    "badaso/content-module": "dev-main",
```

Comment:
don't use dev-main on PR to main branch, because this branch for release

make sure use specific release version @miftahurrahmi